rpm/python-cffi.spec

```diff
@@ -0,0 +1,82 @@
+%bcond_with tests
+%bcond_with doc
+
+Name:           python-cffi
+Version:        1.14.5
+Release:        0
+Summary:        Foreign Function Interface for Python to call C code
+License:        MIT
+URL:            https://cffi.readthedocs.org/
+Source0:        %{name}-%{version}.tar.gz
+
+BuildRequires: make
+BuildRequires:  libffi-devel
+BuildRequires:  gcc
+
+# For tests:
+BuildRequires:  gcc-c++
+
+#?python_enable_dependency_generator
+
+%description
+Foreign Function Interface for Python, providing a convenient and
+reliable way of calling existing C code from Python. The interface is
+based on LuaJIT\u2019s FFI.
+
+%package -n python3-cffi
+Summary:        Foreign Function Interface for Python 3 to call C code
+%if %{with doc}
+BuildRequires:  python3-sphinx
+%endif
+%if %{with tests}
+BuildRequires:  python3-pytest
+%endif
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-pycparser
+%{?python_provide:%python_provide python3-cffi}
```

Comment:
This shouldn't be used anymore. The dependency generator should provide the provides automatically.